addons/account/models/account_move_line.py

```diff
@@ -1335,6 +1335,27 @@ def check_field_access_rights(self, operation, field_names):
             result = [fname for fname in result if fname not in weirdos]
         return result
 
+    @api.model
+    def _get_default_read_fields(self):
+        weirdos = {'term_key', 'epd_key', 'epd_needed', 'discount_allocation_key', 'discount_allocation_needed'}
+        return [fname for fname in self.fields_get(attributes=()) if fname not in weirdos]
+
+    def read(self, fields=None, load='_classic_read'):
+        fields = fields or self._get_default_read_fields()
```

Comment:
Why is this not done at the ORM level?
I think it was the case before?

Reply:
At the ORM level, `read` gets all accessible fields.
Here you don't want to fetch an accessible field; there is only 2 usages like that in accounting, so I added the function here.

`check_field_access_rights` does 2 jobs today: both checking field accesses and getting the defaults.